addons/web/static/src/views/fields/percent_pie/percent_pie_field.xml

```diff
@@ -11,7 +11,9 @@
 
         <div class="o_pie_info">
             <span class="o_pie_value" t-esc="formattedValue + '%'"/>
-            <span class="o_pie_text" t-esc="props.string"/>
+            <t t-if="!props.name.startsWith('x_studio_')">
```

Comment:
As a general guideline refrain from writing code that references modules outside of the dependencies, web doesn't depend on web_studio, there should be no references to web_studio in web.